drivers/growattwithbatt/driver.flow.compose.json

```diff
@@ -1,164 +1,184 @@
 {
   "actions": [
     {
-      "id": "battery_min_cap_load_first",
+      "id": "grid_first_for_interval_with_percentage",
```

Comment:
don't change names or params , it will break existing flows.

Reply:
It is not changed, it is still present in line 123 (although I'm probably the only one who uses it now :) )
It is just GitHub diff does not got the point of the change :)